.github/workflows/test.yml

```diff
@@ -61,6 +61,39 @@ jobs:
     - name: Run root tests
       run: cd libbpf-rs && cargo test --profile=${{ matrix.profile }} --locked --verbose -- test_sudo_
 
+  build-features:
+    name: Build [${{ matrix.args }}]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - args: "features = ['vendored']"
+          - args: "features = ['static']"
+          # TODO: Should build without features, but that requires system
+          #       libbpf and ubuntu 22.04 only has 0.5 (..?)
```

Comment:
yeah, I know :( next LTS is 24.04, right? That will have at least v1.3, though, so just a little more to wait for it...